src/main/java/com/icthh/xm/ms/entity/config/XmEntityTenantConfigService.java

```diff
@@ -59,6 +59,11 @@ public XmEntityTenantConfig getXmEntityTenantConfig() {
         return configs.get(tenantKey);
     }
 
+    public XmEntityTenantConfig getXmEntityTenantConfig(String tenantKey) {
+        configs.computeIfAbsent(tenantKey, (key) -> new XmEntityTenantConfig());
+        return configs.get(tenantKey);
```

Comment:
In accordance with [Map.computeIfAbsent](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#computeIfAbsent-K-java.util.function.Function-) value returned - the current (existing or computed) value, so that `configs.get(tenantKey)` is redundant operation.